tests/gtest/aviflosslesstest.cc

```diff
@@ -64,26 +65,70 @@ void ReadImageSimple(const std::string& file_name, avifPixelFormat pixel_format,
   ASSERT_NE(file_format, AVIF_APP_FILE_FORMAT_UNKNOWN);
 }
 
+// Checks whether an image is grayscale. ASSERT_NO_FATAL_FAILURE should be used
+// when calling it.
+void IsGray(const std::string& path, bool& is_gray) {
+  ImagePtr image(avifImageCreateEmpty());
+  ASSERT_NE(image, nullptr);
+  // In the first iteration, figure out if the image is grayscale.
```

Comment:
I'm not sure what `In the first iteration ...` is referring to. I don't see a loop.

Reply:
Done